torch/_inductor/kernel/mm.py

```diff
@@ -1170,6 +1170,9 @@ def tuned_scaled_mm(
                 epilogue_fn=scale_mm_epilogue(),
             )
 
+    if is_nonzero and use_cutlass_template(layout, m, n, k):
+        CUTLASS3xGemmTemplate.add_cutlass_gemm_choices(choices, layout, input_nodes)  # type: ignore[arg-type]
```

Comment:
maybe somehwere assert use_fast_acumm = False in the mm scaled lowering

Reply:
Yeah I can just check if use_fast_accum = False and not add any choices (and print something)